pom.xml

```diff
@@ -280,9 +281,10 @@
             <module>plugins/jdg64</module>
             <module>plugins/jdg65</module>
             <module>plugins/jdg66</module>
+            <module>plugins/jdg70</module>
          </modules>
          <properties>
-            <plugins.jdg>jdg60,jdg61,jdg62,jdg63,jdg64,jdg65,jdg66</plugins.jdg>
+            <plugins.jdg>jdg60,jdg61,jdg62,jdg63,jdg64,jdg65,jdg66,jdg70</plugins.jdg>
```

Comment:
@alanfx Most likely log4j -> log4j2 transition issue. Please try to build the distribution using -Dlog4j2 property.